scripts/utils/create_package_template_from_json.py

```diff
@@ -25,8 +25,12 @@ def main():
     for k, v in pkg.items():
         if k in ("why", "dependencies", "info"):
             continue
-        cmd_args.append(f"--{k}")
-        cmd_args.append(f"{v}")
+        #the value of arguments may contain - or -- therefore we add --argument=<value>
+        if k == "arguments":	
+            cmd_args.append(f"--{k}={v}")
+        else:
+            cmd_args.append(f"--{k}")
+            cmd_args.append(f"{v}")
```

Comment:
What about always using the `--argument=<value>` format? I think it is in general more robust and it simplifies the code. Or is there any issue with this approach? :thinking: 

```suggestion
        # Use "--argument=<value>" format to avoid issues with if the value contains "-" 
        cmd_args.append(f"--{k}={v}")
```